instruments/abstract_instruments/comm/usb_communicator.py

```diff
@@ -57,10 +94,7 @@ def terminator(self):
     def terminator(self, newval):
         if not isinstance(newval, str):
             raise TypeError("Terminator for USBCommunicator must be specified "
-                            "as a single character string.")
-        if len(newval) > 1:
-            raise ValueError("Terminator for USBCommunicator must only be 1 "
-                             "character long.")
```

Comment:
Probably history, once upon a time we only supported termination strings of length 1, and the PyUSB connector was never developed because of lack of suitable equipment to test it against.

Reply:
Sounds reasonable, this implementation is working w/ the USB device, so I'll leave this removed.